src/components/Button/Button.scss

```diff
@@ -1,28 +1,33 @@
-.btn {
-  padding: 0.6rem 1rem;
-  background: transparent;
-  border: 1px solid #1e1e1e;
-  cursor: pointer;
+@use '../../styles/scss/typography.scss';
+@use '../../styles/scss/paddings.scss';
+@use '../../styles/scss/margins.scss';
+@use '../../styles/scss/colors.scss';
+@use '../../styles/scss/variables.module.scss';
 
-  &-primary {
-    border: none;
-    background-color: blue;
-    color: white;
+.#{variables.$prefix}-btn {
+  border-radius: 0.4rem;
+  padding: paddings.$xs paddings.$sm;
+  margin: margins.$xs margins.$xs;
+
+  @include typography.text-regular();
+
+  &.primary {
+    border: 0.1875rem solid colors.$accent;
```

Comment:
Worth putting `0.1875rem` in local variable `$border-size`

Reply:
Or an even better idea - put `border: 0.1875rem solid colors.$accent;` in `.#{variables.$prefix}-btn` because it is the same for both button variations